src/functionaltests/resources/ThreadingAndWaitlistRules.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
such small json data could be stored in a private static final variable and reduce the need to spend resources on reading from a file.

Reply:
It is changed as suggested

Reply:
Then this file could be removed :)